### PR TITLE
run.sh functions renamed and moved to utility_functions.sh

### PR DESCRIPTION
**Pull request by jdziurzynski**
KNT-243

run(),set_test_env() and check_env() functions located in examples/../run.sh scripts moved to one file located in tools/utility_functions.sh and renamed (suffix _examples)

### CODE REVIEW

**Review by mmac-m3a — APPROVED** [review]

File: tools/utility_functions.sh [hunk 1]
```diff
@@ -49,3 +49,46 @@ function colored_echo() {
...
+    pushd "$SCRIPT_DIR" > /dev/null || die
+    rm -rf .terraform .terraform.lock.hcl
+
+    terraform init || die
+    terraform apply
+
+    popd > /dev/null || die
+}
+
+function set_test_env_examples() {
```
Comment (danielfurman) [inline tools/utility_functions.sh:79]:
```suggestion
function set_test_env() {
```

Comment (danielfurman) [reply tools/utility_functions.sh:79]: set_test_env is self-explanatory

File: tools/utility_functions.sh [hunk 2]
```diff
@@ -49,3 +49,46 @@ function colored_echo() {
...
+    export KTAPI_URL="http://$TEST_API_SERVER_ENDPOINT"
+    export KTAPI_AUTH_EMAIL="dummy@acme.com"
+    export KTAPI_AUTH_TOKEN="dummy"
+}
+
+function check_env_examples() {
```
Comment (danielfurman) [inline tools/utility_functions.sh:85]:
```suggestion
function check_credentials_in_env() {
```
More informative alternative

**Review by danielfurman — COMMENTED** [review]
Added renaming suggestions

**Review by danielfurman — APPROVED** [review]

**Review by danielfurman — COMMENTED** [review]
I still think names could be improved

**Review by mmac-m3a — APPROVED** [review]

**Pull request merged**